pkg/fanal/cache/fs_test.go

```diff
@@ -225,6 +225,7 @@ func TestFSCache_PutBlob(t *testing.T) {
 				        {
 				          "Name": "musl",
 				          "Version": "1.1.22-r3",
+						  "Identifier": {},
```

Comment:
Looks like we need to update this test

Reply:
done at https://github.com/aquasecurity/trivy/pull/5439/commits/59b1198b98913b617257772c5233450ffe9e3b5c